### PR TITLE
Merge master jdk-23.0.1+11 into openj9-staging

### PR DESCRIPTION
**Pull request by JasonFengJ9**
Merge master jdk-23.0.1+11 into openj9-staging

Conflicts:
```
	test/jdk/security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java
```

Modified:
```
	test/jdk/ProblemList.txt
```

Signed-off-by: Jason Feng <fengj@ca.ibm.com>

### CODE REVIEW

File: test/jdk/ProblemList.txt [hunk 1]
```diff
@@ -470,9 +470,6 @@ sun/java2d/DirectX/OnScreenRenderingResizeTest/OnScreenRenderingResizeTest.java
 java/awt/Window/8159168/SetShapeTest.java 8274106 macosx-aarch64
 java/awt/image/multiresolution/MultiResolutionJOptionPaneIconTest.java 8274106 macosx-aarch64
 
-# This test fails on macOS 14
-java/awt/Choice/SelectNewItemTest/SelectNewItemTest.java 8324782 macosx-all
-
```
Comment (keithc-ca) [inline test/jdk/ProblemList.txt:475]: This file should be identical to 9efecc96faa6d0b52a4555ee036c4dce6f4658a3.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: test/jdk/ProblemList.txt (hunk 1 above)
Comment (JasonFengJ9) [reply test/jdk/ProblemList.txt:475]: https://github.com/ibmruntimes/openj9-openjdk-jdk23/commit/9efecc96faa6d0b52a4555ee036c4dce6f4658a3 was following a few changes https://github.com/openjdk/jdk23u/commits/master/test/jdk/ProblemList.txt

GIT missed the latest changes https://github.com/ibmruntimes/openj9-openjdk-jdk23/pull/58#discussion_r1803815531 when merging the current OpenJDK contents to `master` branch.
The change is made manually in this PR to keep `openj9-staging` consistent with OpenJDK updates.

Comment (keithc-ca) [reply test/jdk/ProblemList.txt:475]: Those subsequent changes are from commits that are not ancestors of [jdk-23.0.1+11](https://github.com/ibmruntimes/openj9-openjdk-jdk23/commit/df073d9f7e95611bf43d61c70add424b9bc3b9a7) and so don't belong in this pull request which claims to merge up to that tag.

Comment (JasonFengJ9) [reply test/jdk/ProblemList.txt]: Restored [jdk-23.0.1+11](https://github.com/ibmruntimes/openj9-openjdk-jdk23/commit/df073d9f7e95611bf43d61c70add424b9bc3b9a7) content.

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**